client/lib/human-date/index.js

```diff
@@ -1,28 +1,24 @@
-import i18n from 'i18n-calypso';
-import moment from 'moment';
-
+import { useTranslate } from 'i18n-calypso';
+import { useMemo, useState } from 'react';
+import { useLocalizedMoment } from 'calypso/components/localized-moment';
+import { EVERY_TEN_SECONDS, useInterval } from 'calypso/lib/interval';
 const MILLIS_IN_MINUTE = 60 * 1000;
 
-export default function humanDate(
-	dateOrMoment,
-	dateFormat = 'll',
-	locale = i18n.getLocaleSlug()
-) {
-	const now = moment().locale( locale );
-	dateOrMoment = moment( dateOrMoment ).locale( locale );
+function getHumanDateString( now, date, dateFormat, moment, translate ) {
+	date = moment( date );
```

Comment:
There should be also a `now = moment( now )` line, otherwise calls to `now.diff( date )` will fail?

Reply:
Oops, yes ofc 👍🏻